app/models/special_enrollment_period.rb

```diff
@@ -238,6 +251,17 @@ def set_submitted_at
     self.submitted_at ||= TimeKeeper.datetime_of_record
   end
 
+  def set_coverage_renewal_flag
+    prior_py_ivl_sep = EnrollRegistry.feature_enabled?(:prior_plan_year_ivl_sep)
+    return if coverage_renewal_flag == false || prior_py_ivl_sep == false
+
+    self.assign_attributes({coverage_renewal_flag: prior_py_ivl_sep})
+  end
+
+  def set_user_id
+    self.user_id = SAVEUSER[:current_user_id]
```

Comment:
Looks like user_id is not saving, can you check after SEP record in DB.

Reply:
Will look into it